src/ViewRendererBundle/objects/helper/TPkgViewRendererLessCompiler.php

```diff
@@ -139,15 +129,15 @@ public function getCompiledCssFilenamePattern()
      */
     protected function getLessFilesFromSnippetsAndTheme($portal)
     {
-        $resourceCollector = new TPkgViewRendererSnippetResourceCollector();
-        $resources = $resourceCollector->getLessResources($portal, CMS_SNIPPET_PATH);
-
         $theme = null;
-        if (is_object($portal)) {
+        if (null !== $portal) {
```

Comment:
here we want to get the theme from the theme service - not from the portal. Otherwise we have no way of returning the less files for the backend theme.

Reply:
Hm, also this location expects everything to be "frontend". So when no explicit additional less file is configured for this theme "/assets/less/chameleon.less" is loaded. And that is normally used for frontend less.

Reply:
The service that provides the theme will work here, will it not? If so, then please do use it.

Reply:
Done